cmake/msvc_static_runtime.cmake

```diff
@@ -0,0 +1,8 @@
+FOREACH (flag_var
+    CMAKE_C_FLAGS CMAKE_C_FLAGS_DEBUG CMAKE_C_FLAGS_RELEASE
```

Comment:
Instead of hardcoding configurations, could you please use CMAKE_CONFIGURATION_TYPES ?